fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkcommon/BulkDataUtils.java

```diff
@@ -84,8 +85,10 @@ public static AmazonS3 getCosClient(String cosCredentialIbm, String cosApiKeyPro
             credentials = new BasicAWSCredentials(cosApiKeyProperty, cosSrvinstId);
         }
 
-        ClientConfiguration clientConfig = new ClientConfiguration().withRequestTimeout(8000);
-        clientConfig.setUseTcpKeepAlive(true);
+        ClientConfiguration clientConfig = new ClientConfiguration()
+                .withRequestTimeout(10*1000)
+                .withTcpKeepAlive(true)
+                .withSocketTimeout(120*1000);
```

Comment:
maybe convert the values to constants explaining REQUEST_TIMEOUT_IN_SECONDS or something similar -- 1000 = second

Reply:
good idea!

Reply:
done